lib/chef/knife/bmcs_server_delete.rb

```diff
@@ -29,16 +29,38 @@ class BmcsServerDelete < Knife
              long: '--wait SECONDS',
              description: 'Wait for the instance to be terminated. 0=infinite'
 
+      option :prune,
+             long: '--prune',
+             description: 'Also remove node from Chef server.'
```

Comment:
Should mention here that the display name is used unless --chef-node-name is specified.